documentation/reference/services-content.html

```diff
@@ -387,6 +387,8 @@ <h2 id="group">group</h2>
 Can not be used in conjunction with the <code><a href="#nodes">nodes</a></code> element.
 If a non-flat structure is desired, use this element instead.
 Groups can contain other groups or nodes, but not both.
+When using groups, <code><a href="#searchable-copies">searchable-copies</a></code> and
```

Comment:
There are two ways to configure groups, one for cloud and one for self-hosted. In the cloud case, redundancy, final redundancy and searchable-copies is per group, while in the self-hosted case it's across all groups. Perhaps we need to say something about both here now.

Reply:
Something like the change below?

Reply:
you can alternatively add to https://cloud.vespa.ai/reference/services-hosted.html and link to this from the open source doc

Reply:
I think it might be confusing to make the distinction be between "open source", since it's the same open source code base in both cases .... we're going to have a thousand of these so we better get it right. I suggest "when self-hosting"/"in self-hosted systems" vs. "cloud"/"Vespa cloud".